README

```diff
@@ -1,5 +1,9 @@
-GREG Please move this info to the associated examples.
+GREG:
 
+1) Please move this info into the associated examples.
+2) Look at rdbService and serviceAP
+I left these almost untouched from release 4.5
+3) Look at KNOWNPROBLEMS.md
```

Comment:
I am only referring to this very paragraph:

 > GREG:
 > 1) Please move this info into the associated examples.
 > 2) Look at rdbService and serviceAP
 > I left these almost untouched from release 4.5
 > 3) Look at KNOWNPROBLEMS.md

I think this text should be an issue in this module.
It doesn't cover other modules as far as I can see, has clear instructions, and is directed to a single person.
So - by its nature - it is an issue that is assigned to someone, rather than a bullet in a list of known problems.